README.md

```diff
@@ -97,6 +97,115 @@ In cases where this default variable wrapping can cause problems (e.g. a variabl
 
   `{{#wrap}}[{{nowrap link.text}}]({{nowrap link.url}}){{/wrap}}`
 
+#### Handlebars helpers
+
+In addition to the standard helpers which are part of Handlebars, there are a number of additional helpers defined to assist with Ricardian contract specific issues.
+
+___Since v0.0___
+* `wrap` - Takes no parameters. Block level helper to wrap the contents with `<div>` tags for highlighting. See [Variables](#Variables) section above for more details.
+
+* `nowrap` - Takes a variable. Indicates that the specified variable *should not* be wrapped with `<div>` tags. See [Variables](#Variables) section above for more details.
+
+* `symbol_to_symbol_code` - Given a variable containing a 'symbol' string, extract and return only the symbol code.
+
+  Example:
+  ```
+  Given:
+  symbol = '4,EOS'
+  
+  {{ symbol_to_symbol_code symbol }} --> 'EOS'.
+  ```
+
+* `asset_to_symbol_code` - Given a variable containing an 'asset' string, extract and return only the symbol code.
+
+  Example:
+  ```
+  Given:
+  asset = '2.001 EOS'
+  
+  {{ asset_to_symbol_code asset }} --> 'EOS'.
+  ```
+
+___Since v0.1___
+* `account_in_permission_level` - _(__Deprecated__ Unnecessary since the account name can be accessed directly on the authorization object via standard dot notation.)_ Given a variable containing a permission level object, extract and return the account name.
+
+  Example:
+  ```
+  Given Permission level / authorization:
+  auth = {
+          "actor": "alicejones",
+          "permission": "active"
+        }
+
+  In Ricardian contract:
+  {{ account_in_permission_level auth }} --> 'alicejones'
+  ```
+
+* `permission_in_permission_level` - _(__Deprecated__ Unnecessary since the permission name can be accessed directly on the authorization object via standard dot notation.)_ Given a variable containing a permission level object, extract and return the permission name.
+
+  Example:
+  ```
+  Given Permission level / authorization:
+  auth = {
+          actor: 'alicejones',
+          permission: 'active'
+        }
+
+  In Ricardian contract:
+  {{ permission_in_permission_level auth }} --> 'active'
+  ```
```

Comment:
Indentation could be improved here.

Reply:
Because these two helpers are being documented _for the first time_ and they're deprecated, can we just drop them?